roles/zuul/templates/etc/zuul/config/layout.yaml

```diff
@@ -21,10 +21,12 @@ pipelines:
     success:
       github:
         status: true
+        status_url: &log_link https://logs.bonnyci.com/{pipeline.name}/{change.project.name}/{change.number}/{item.enqueue_time}
```

Comment:
I feel like the "logs.bonnyci.com" bit needs to be a template value, if this were to be useable by other orgs for other URLs. Not a huge issue for now though.